docs/csslint-examples.md

```diff
@@ -59,6 +59,26 @@ csslint: {
 }
 ```
 
+## Providing a custom formatter
+
+```js
+csslint: {
+  options: {
+    csslintrc: '.csslintrc',
+    formatter: require('csslint-stylish),
+    formatters: [
+      {id: 'stylish', dest: 'report/csslint_stylish.xml'}
```

Comment:
Ok, fixed (and tested). Not sure I like the code though :laughing:. Ideas for improvement?